### ISSUE TITLE
MC 1-12-2 Forge 2611 and woot-1.12.2-1.0.0 Tier IV Enderdragon Heart Issues

### ISSUE THREAD

**Issue by trogarus** [Ipsis/Woot#328]
For some reason it's not giving the heart as a drop. Did I miss something? I get the egg, draconic dust, and dragon breath, but no heart.

**Comment by Ipsis** [comment]
That particular drop is not guaranteed. You can see the chance in the gui if you hover over it.
I defaulted the chances to be the same as ATM3 modpack, which may have used some rather cruel values.

"sizes": [ 1, 1, 1, 1 ],
"chances": [ 20, 40, 60, 80 ]

**Comment by trogarus** [comment]
Yeah, deviates from the vanilla mechanic which is a guaranteed drop. I'll look about the json and figure out how to put it back to 100% drop. Thanks, just looked odd. =)

**Comment by Ipsis** [comment]
Note to myself: Reorder the chances to allow guaranteed at Looting III
eg.
"chances": [ 25,50,75,100]

**Comment by trogarus** [comment]
lol, thanks for the heads up on the json btw, found it, set it all 100 percent =) Thanks again for your work on the mod and setting those, like me, straight when we miss something =)

**Comment by Ipsis** [comment]
I'm going to update this in my configs as well, since as you pointed out, it deviates from Draconic mechanics. The values from ATM that I used are a bit much. Do I'll keep this issue open until I've fixed that.

**Comment by trogarus** [comment]
I will give you a bit of advice when it comes to mechanics and "balance". Firstly, with "balance", it means different things to different people. If you mod developers really want to free yourselves from the "angry mobs" or "vocal majority", then make sure you have configs that allow people to do their own version of "balance" or "mechanics". You have accomplished that through this method, so don't sweat the numbers. They can easily change them for themselves in the configs.

Secondly, in doing so, you free yourself up to devote more time to your mod, and less time trying to do things that the users can then do for themselves. I've never understood why this is such a complex idea to many mod devs. It's usually better to give users choices and allow them to set those choices on their own. Less people then able to take up your time complaining about things they don't like with your chosen "balance" or mechanics.

On final note, I appreciate your candor and the way you're open to things. I've met many mod developers lacking in this area of maturity. Thanks again for all your hard work on this mod. It is greatly appreciated, at least by this user.

**Comment by OrdinatorStouff** [comment]
I am flattered by you referencing ATM defaults but as you yourself already were aware they are probably more "cruel" than expected. Then again, that's a matter of opinion anyway. Woot is a "set-it-and-forget-it" mod. Build the multiblock, feed it RF and required items and come back 20 minutes later or 20 hours later to your hoards of loot.   
  
 [ 20, 40, 60, 80 ] for dragon heart is not taken in perspective of a single dragon kill. I think that taking the time to go to the end, place your 4 end crystals, wait for the dargon to respawn for like 30 seconds and then hunt/kill her manually warrants a "guaranteed" heart. As soon as you have a woot multiblock programmed to her and have it basically "on farm" then the percentages assocaited with looting are less about the actual dargon fight and more about the infinite dragon-egg-and-heart farm you just got up and running. 
  
At least in the context of ATM3 at least. Not everyone will agree, not every situation will have the same perspective.
